david-davis.txt

```diff
@@ -0,0 +1,106 @@
+
+https://codepen.io/DCdavis/pen/qBEyrGQ?editors=1100
+
+    1. What is Semantic HTML? 
+    HTML is the structure on the page.
+
+    2. What is HTML used for?
+    HTML is used to build the page.
+
+    3. What is an attribute and where do we put it? 
+    attributes are used in opening tags to control an elements behavior.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    The h1 tag is used for the most important heading on the page and they should only be used once.
+
+    5. Name two tags that have required attributes
+      nav and img both use attributes.
+
```

Comment:
`<img>` is a tag that requires the `src` attribute so that's spot on. Does nav require a certain attribute as well?